vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/bdd/steps/integration/HttpRequestSteps.java

```diff
@@ -16,23 +16,21 @@
 
 package org.vividus.bdd.steps.integration;
 
-import java.util.stream.Collectors;
+import java.util.Map;
 
 import javax.inject.Inject;
 
 import org.apache.http.client.CookieStore;
-import org.apache.http.cookie.Cookie;
 import org.jbehave.core.annotations.When;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.vividus.http.HttpTestContext;
+import org.vividus.reporter.event.AttachmentPublisher;
 import org.vividus.ui.web.action.ICookieManager;
 
 public class HttpRequestSteps
 {
-    private static final Logger LOGGER = LoggerFactory.getLogger(HttpRequestSteps.class);
     @Inject private ICookieManager cookieManager;
     @Inject private HttpTestContext httpTestContext;
+    @Inject private AttachmentPublisher attachmentPublisher;
```

Comment:
what about changing to constructor?

Reply:
are we going to completely migrate from field injection to constructor injection?

Reply:
yes, I think